content/blogposts/blog-post-starter-pack.md

```diff
@@ -0,0 +1,261 @@
+
+
+### Blockquote
+
+Include a `Blockquote` to highlight a key quote. Optionally, you can also provide context above the quote with a `headline`. 
+
+<Blockquote
+  headline="Search your code. All of it."
+  quote="Onboard to a new codebase, find answers faster, and identify security risks with universal code search."
+  author="Sourcegraph"
+/>
+
+
+### Figure
+
+Images help tell your story. Add them to your blog posts with the `Figure` component. If you would like to include a description, use the `caption` property. This will automatically be styled, so there is no need to include a separate `figcaption` description below an image.
+
+<Figure 
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/default_hero_social.png"
+  alt="Sourcegraph thumbnail"
+  caption="The Sourcegraph logo"
+/>
+
+
+### TableWrapper
+
+Tables are a straightforward way to present and compare data. When preparing a table, please include the `TableWrapper` component to ensure that the table adapts to different screen sizes. To support your preference, our blog accepts tables written in both Markdown and HTML.
+
+This table is written in **Markdown**:
+
+<TableWrapper>
+| Id   | Species     | Sepal Length (cm) | Sepal Width (cm) | Petal Length (cm) | Petal Width (cm) |
+|------|-------------|-------------------|------------------|-------------------|------------------|
+| 1    | Iris Setosa | 5.1               | 3.5              | 1.4               | 0.2              |
+| 2    | Iris Setosa | 4.9               | 3.0              | 1.4               | 0.2              |
+</TableWrapper>
+
+This is the same table, but written in **HTML**:
+
+<TableWrapper>
+  <table>
+    <thead>
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Sepal Length (cm)</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>1</td>
+        <td>Iris Setosa</td>
+        <td>5.1</td>
+        <td>3.5</td>
+        <td>1.4</td>
+        <td>0.2</td>
+      </tr>
+      <tr>
+        <td>2</td>
+        <td>Iris Setosa</td>
+        <td>4.9</td>
+        <td>3.0</td>
+        <td>1.4</td>
+        <td>0.2</td>
+      </tr>
+    </tbody>
+  </table>
+</TableWrapper>
+
+
+### YouTube
+
+You can easily embed YouTube videos using the `YouTube` component. The `id` value is found at the end of the URL for any YouTube video. To display the title below the video, add the `showTitle` property.
+
+<YouTube
+  title="Accelerate developer onboarding with Sourcegraph" 
+  id="DgwvhRW1Cbc"
+  showTitle={true}
+/>
+
+
+### Video
+
+The `Video` component supports .mp4 and .webm files. First, upload the two versions of the video to Google Cloud. Then, add the root names of these files to the `source` property, without including their extensions. You can choose to repeat a video by including the `loop` property, or display the caption using the `showCaption` property.
```

Comment:
The API will change slightly here in #5615

```suggestion
The required properties for the `Video` component are `mp4`, `webm`, `loop`, `caption`, and `title`. 

Since the component supports both `.mp4` and `.webm` files, you'll need to upload the two versions of the video to our Google Cloud [assets bucket](https://console.cloud.google.com/storage/browser/sourcegraph-assets/blog) under an appropriate sub directory. Then, add the file path from the Public URL of these files to the `mp4` and `webm` properties without their file extensions.

You can choose to repeat a video by including the `loop` property, optionally auto play the video by including the `autoPlay` property, or optionally display the caption using the `showCaption` property.
```